### ISSUE TITLE
Add option for shaded area to intercept and ignore all interactions.

### ISSUE THREAD

**Issue by shirakaba** [Tawa/TNTutorialManager#2]
(As titled). Effectively, this would allow actions only to occur through the highlighted area.

**Edit:** This would need to be optional *per stage* of the tutorial. Example use case:

* In the first tutorial stage, a label saying "Click anywhere to begin tutorial" is shown. This stage would accept a tap action regardless of the position tapped on the screen (as standard).

* In the next stage, a highlight appears around a button; the user must press within this area (ideally, a genuine tap of the button should be required, but we can create the illusion of it programmatically, so it's not a big problem) to continue; any taps elsewhere would not advance the tutorial to the next stage.

**Comment by shirakaba** [comment]
Suggested interface:

```Objective-C
/**
 This optional method returns whether, for any given index, the tutorial should advance only upon taps made INSIDE the highlighted area (TRUE) rather than anywhere in the whole tutorial area (FALSE; default value).
 */
-(BOOL)tutorialRestrictActionsToWithinHighlight:(NSInteger)index;
```

**Comment by Tawa** [comment]
This was fixed in version 1.3! Please see updated README file.
